packages/react/src/data-table/DataTable.css.ts

```diff
@@ -0,0 +1,61 @@
+import { createVar, recipe, style } from "../vanilla-extract";
+
+export const columnWidth = createVar();
+export const cellOffset = createVar();
```

Comment:
vars should have the suffix `Var` to clearly denote they are css vars
```suggestion
export const columnWidthVar = createVar();
export const cellOffsetVar = createVar();
```